matrix.yml

```
@@ -3,6 +3,7 @@ ansible-core:
   - 2.12.10
   - 2.13.5
   - 2.14.0b3
+  - 2.14.0rc1
```

With the rc1 we don't need to test the b3 anymore
```suggestion
  - 2.14.0rc1
```

---

good, will push...